packages/webapp/src/routes/__tests__/land/check-land-boundary-details.spec.js

```diff
@@ -16,10 +16,40 @@ describe(url, () => {
       sessionData[`${constants.redisKeys.FULL_NAME}`] = 'Test User'
       sessionData[`${constants.redisKeys.ROLE_KEY}`] = 'test'
       sessionData[`${constants.redisKeys.EMAIL_VALUE}`] = 'test@example.com'
+      sessionData[`${constants.redisKeys.LAND_BOUNDARY_GRID_REFERENCE}`] = 'SE170441'
+      sessionData[`${constants.redisKeys.LAND_BOUNDARY_HECTARES}`] = 2
       sessionData[`${constants.redisKeys.LAND_BOUNDARY_UPLOAD_TYPE}`] = 'geospatialData'
+      sessionData[`${constants.redisKeys.LAND_BOUNDARY_LOCATION}`] = '800376c7-8652-4906-8848-70a774578dfe/land-boundary/legal-agreement.doc'
+      sessionData[`${constants.redisKeys.LAND_BOUNDARY_FILE_SIZE}`] = 0.01
+      sessionData[`${constants.redisKeys.LAND_BOUNDARY_FILE_TYPE}`] = 'application/msword'
+      sessionData[`${constants.redisKeys.LAND_BOUNDARY_CHECKED}`] = 'yes'
```

Comment:
do you really need all this test data, if you are not using it? 

I think I don't mind !